doc/modules/clustering.rst

```diff
@@ -392,22 +392,22 @@ for centroids to be the mean of the points within a given region. These
 candidates are then filtered in a post-processing stage to eliminate
 near-duplicates to form the final set of centroids.
 
-Given a candidate centroid :math:`x_i` for iteration :math:`t`, the candidate
+Given a candidate centroid :math:`x` for iteration :math:`t`, the candidate
```

Comment:
We can then mention that the update is a hill-climbing strategy.

Reply:
Good idea